### PR TITLE
feat(core): Update pubsub messages to use 'stream' instead of 'doc'

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Review by oed — CHANGES_REQUESTED** [review]
This is a protocol breaking change. I think we should wait with this until after ELP and think about a real upgrade path.

**Comment by stbrody** [comment]
> This is a protocol breaking change. I think we should wait with this until after ELP and think about a real upgrade path.

Well I did include backwards compatibility logic in this, but I guess there's no reason this has to happen now.  We could wait and do this as part of the protocol versioning project as the first live protocol message format update.

**Comment by oed** [comment]
> Well I did include backwards compatibility logic in this

Did you forget to push this? Definitely no there afaict.

**Comment by stbrody** [comment]
> > Well I did include backwards compatibility logic in this
> 
> Did you forget to push this? Definitely no there afaict.

It's there.  Check out the `serialize` and `deserialize` functions in pubsub-message.ts

File: packages/core/src/pubsub/pubsub-message.ts [hunk 1]
```diff
@@ -64,7 +64,8 @@ export function serialize(message: PubsubMessage): string {
     case MsgType.QUERY: {
       return JSON.stringify({
         ...message,
-        doc: message.doc.toString(),
+        doc: message.stream.toString(), // todo remove once we no longer support interop with nodes older than v1.0.0
+        stream: message.stream.toString(),
```
Comment (oed) [inline packages/core/src/pubsub/pubsub-message.ts:68]: 👀

**Review by oed — APPROVED** [review]
Sneaky! Alright my bad. This should be fine then :)

**Pull request merged**